src/functions/Coverage.ps1

```diff
@@ -1049,6 +1049,225 @@ function Get-JaCoCoReportXml {
     return $xml
 }
 
+function Get-CoberturaReportXml {
+    param (
+        [parameter(Mandatory = $true)]
+        [object] $CoverageReport,
+        [parameter(Mandatory = $true)]
+        [long] $TotalMilliseconds
+    )
+
+    if ($null -eq $CoverageReport -or $CoverageReport.NumberOfCommandsAnalyzed -eq 0) {
+        return [string]::Empty
+    }
+
+    $now = & $SafeCommands['Get-Date']
+    $nineteenSeventy = & $SafeCommands['Get-Date'] -Date "01/01/1970"
+    [long] $endTime = [math]::Floor((New-TimeSpan -start $nineteenSeventy -end $now).TotalMilliseconds)
+    [long] $startTime = [math]::Floor($endTime - $TotalMilliseconds)
+
+    $commonRoot = Get-CommonParentPath -Path $CoverageReport.AnalyzedFiles
+
+    $allLines = [System.Collections.Generic.List[object]]@()
+    $null = $allLines.AddRange($CoverageReport.MissedCommands)
+    $null = $allLines.AddRange($CoverageReport.HitCommands)
```

Comment:
```suggestion
    $allLines.AddRange($CoverageReport.MissedCommands)
    $allLines.AddRange($CoverageReport.HitCommands)
```

IIRC this does not output, because only ArrayList does that